### ISSUE TITLE
[BUG]Unable to connect Ledger Wallet to Liquidity Mining app

### ISSUE THREAD

**Issue by RC-13** [CoinAlpha/hummingbot#1733]
**Describe the bug**

Failed to connect or register a Ledger wallet on the liquidity miner app. Upon connecting the Ledger wallet, the eth address will be shown for a couple of seconds and disappear, wallet textbox by then will be disabled.
Users ledger uses firmware version 1.6.0 and Ethereum app is 1.2.13

**Steps To Reproduce**
1. Login to your Ledger and open ETH app
2. Click "Connect Wallet" button at the top right navigation of the miner app page
3. Select "Ledger" to connect your Ledger
![connect ledger](https://user-images.githubusercontent.com/57189990/81775860-3215c100-9520-11ea-9bc7-ba3f5d9ff76b.png)
4. A window will appear with a detailed guide on how to connect to your Ledger. 
5. The eth wallet will be displayed for a couple of seconds then disappear and the wallet textbox will be disabled.
![Ledger](https://user-images.githubusercontent.com/57189990/81775981-802ac480-9520-11ea-86ef-c26a209a6820.png)

Note: The user tried using Arch Linux and Windows OS with the same result.
**Release version**
// Include your bot's version number (Can be found at the upper left corner of your CLI).

**Attachments**
// Please attach your config file and log file to help us debug. It would be difficult for us to help you without those! 

*WARNING: Do NOT publish any exchange API keys or your wallet's private key. Whoever has access to them may steal your assets!*

**Optional: your discord username:**
Stefan

**Comment by devoger** [comment]
firmware version 1.6.0 and Ethereum app is 1.2.13
![Screen Shot 2020-05-13 at 18 07 01](https://user-images.githubusercontent.com/8411294/81799979-b37f4a80-9544-11ea-8264-1ebf49540d02.png)


I tested using firmware version 1.6.0 and Ethereum app is 1.2.12 and it is working just fine.
Browser Brave
![ledgernano1](https://user-images.githubusercontent.com/8411294/81798317-79ad4480-9542-11ea-96b8-d66d74dbe1f1.png)


I tested using firmware version 1.6.0 and Ethereum app is 1.2.13. 
Browser: Firefox and Chrome
I reproduced below error after confirming sign message:
![Screen Shot 2020-05-13 at 17 40 17](https://user-images.githubusercontent.com/8411294/81798527-baa55900-9542-11ea-891f-a2f735fd828c.png)

I tried Ethereum app is 1.2.13. 
Browser Brave and it is working.
![Screen Shot 2020-05-13 at 17 55 42](https://user-images.githubusercontent.com/8411294/81799569-18867080-9544-11ea-8fd0-862272bdb6a1.png)
